src/main/java/jp/ac/keio/bio/fun/xitosbml/CuiRun.java

```diff
@@ -0,0 +1,52 @@
+package jp.ac.keio.bio.fun.xitosbml;
+
+import java.util.concurrent.Callable;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "Running XitoSBML-CUI", mixinStandardHelpOptions = true, description = "Saves SBML document and spatial image", version = "1.0")
+
+public class CuiRun implements Callable<Integer> {
+
+	@Option(names = "-i", required = true, description = "The input option")
+	String iValue;
+	@Option(names = "-o", required = true, description = "The output option")
+	String oValue;
+
+	@Override
+	public Integer call() {
+		System.out.printf("-i=%s%n", iValue);
+		System.out.printf("-o=%s%n", oValue);
```

Comment:
Do we need this? Was this for a debugging purpose?

Reply:
Yes, I kept it only for debugging purposes. Will comment it out.

Reply:
Commented these lines out in new branch xitosbml_cui.